data/nlu/niceties.md

```diff
@@ -0,0 +1,109 @@
+## intent:ask_how_contribute
+- How to contribute
+
+## intent:ask_whatspossible
+- What can you do?
+
+## intent:enter_data
+- fe
+- /restart
```

Comment:
All of these are already in the other ones right? I assume they're part of an IL dump?